app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "AllAboutOlaf",
+  "displayName": "All About Olaf"
+}
```

Comment:
I'm still not entirely clear on the purpose of this file – I think it's optional, but the upgrade added it. We'll see what it's used for in the future.